### PR TITLE
index.rst: Make short description short

### PR DESCRIPTION
**Pull request by techtonik**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -7,10 +7,8 @@
 The Mingwpy project
 ###################
 
-The mingwpy project provides a compiler for Python extension modules that is
-based on the :term:`mingw-w64` :term:`gcc` compilers, but where the compiled
-extensions are compatible with Python and other extensions compiled using
-:term:`Microsoft Visual C++`.
```
Comment (ogrisel) [inline index.rst:13]: The original description is a bit more verbose but also more exact. mingwpy's gcc compiler is based on mingw-w64 with significant changes.

Comment (ogrisel) [inline index.rst:13]: Also I think it would be important to tell that this is a C/C++/Fortran compiler toolchain.

Comment (techtonik) [inline index.rst:13]: New description mentions `mingw-w64`.

**Comment by carlkl** [comment]
Hi,

this is too short and not correct:

Mingw-w64: http://mingw-w64.org/doku.php is set set of headers and libraries. Many gcc toolchains on windows are based on the mingw-w64 project and declare themself as mingw64 toolchain or similar. 

However, feel free to improve this summary.

**Comment by carlkl** [comment]
lets include parst of the text from https://github.com/mingwpy/mingwpy/blob/master/README.md instead

**Comment by techtonik** [comment]
Done with some edits and inclusion of "open source" clause that is also important for security specialists. Speaking of benefits I also found that using `gdb` on Windows appeared to be very useful for troubleshooting various C++ related crashes.

**Pull request merged**